boards/avr-rss2/Makefile.features

```diff
@@ -0,0 +1,19 @@
+
+CPU = atmega256rfr2
+
+# This board is based on an atmega CPU, thus import the features from it
+include $(RIOTBOARD)/common/arduino-atmega/Makefile.features
```

Comment:
Is this board a clone of an Arduino ATmega ? If no, I would just remove this line.